specification/paths/Shipping-v2-available-delivery-dates.json

```diff
@@ -38,6 +39,12 @@
                     "description": "The service code for which to fetch the available delivery date time windows.",
                     "example": "carrier-parcel-next-day"
                   },
+                  "carrier_code": {
```

Comment:
Let's position `carrier_code` as the first attribute, because the `service_code` depends on this value